sickbeard/subtitles.py

```diff
@@ -315,7 +315,7 @@ def download_best_subs(video_path, subtitles_dir, release_name, languages, subti
                 logger.debug(u'Could not search in %s provider. Discarding for now', provider)
 
         if not subtitles_list:
-            logger.info(u'No subtitles found for %s', video_path)
+            logger.info(u'No subtitles found for %s', os.path.basename(video_path))
```

Comment:
Wouldn't know why, video_path is always the full path to the video the subtitle is searched for.